### PR TITLE
Recover a deleted microsoft exchange mailbox

### PR DESCRIPTION
**Pull request by WilliamLoy**
recoverhexboxsc1

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4c9a5cc8bc/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-7b39a6259b/how-to/)

File: content/exchange/recover-a-deleted-microsoft-exchange-mailbox.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+### Prerequisites
+
+- **Applies to:** Administrator
+- **Difficulty:** Easy
+- **Time needed:** At least 24 hours for the deleted Microsoft Exchange mailbox to recreate
+- **Tools required:** Cloud Office Control Panel access
+
+For more information about prerequisite terminology, see [Cloud Office support terminology](/how-to/cloud-office-support-terminology).
+
+### How to recover a deleted Microsoft Exchange mailbox
```
Comment (stephamon) [inline content/exchange/recover-a-deleted-microsoft-exchange-mailbox.md]: Recover a deleted Exchange mailbox

File: content/exchange/recover-a-deleted-microsoft-exchange-mailbox.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+**Warning:** A mailbox can only be recovered within 14 days of its deletion.
+
+1.	Log in to the [Cloud Office Control Panel](https://cp.rackspace.com/) using your Rackspace Cloud Office admin ID and password.
```
Comment (stephamon) [inline content/exchange/recover-a-deleted-microsoft-exchange-mailbox.md]: Lot of extra spaces after the number here and in number 2

**Comment by stephamon** [comment]
👍

**Pull request merged**